### DISCUSSION TITLE
Incorrect audio pan with Background Music disabled

### DISCUSSION THREAD

**Discussion by qaisjp** [kyleneideck/BackgroundMusic#494] [Q&A] [answered]
When I have Background Music turned on, all my audio is fine.

However, when Background Music is exited (or uninstalled!), Discord voice audio, and only **Discord's** _voice_ audio (not notifications and other SFX) is panned so that the audio only comes out of one speaker.

This might have been caused by my upgrade to Big Sur a while ago? 

0.4.0-SNAPSHOT-b38f6dd
macOS 11.2.3

I've been ignoring the issue for a while but I'm finally reporting it in the hopes that you know of a solution :D

**Comment by kyleneideck** [comment, marked as answer]
I can't think of anything that could be causing that. The way Background Music applies its pan settings is by modifying the audio stream as it comes through. It doesn't change anything that could change the audio pan when you're not playing the audio through the "Background Music" virtual audio device.

Actually, with SIP enabled, I don't think it could if it wanted to. Apart from just changing the global pan (balance) settings, of course.

Did you uninstall by running `uninstall.sh`? And did it successfully remove the Background Music device from your system?

**Reply by qaisjp** [reply]
My bad. Looks like it's intentional:

- https://twitter.com/discord/status/734064539176710144
- https://www.reddit.com/r/discordapp/comments/6gu0gs/discord_only_coming_out_of_right_speaker/dityhty/
- https://discussions.apple.com/thread/492017?answerId=2383938022#2383938022

> The left speaker is muted because your mic is in the left speaker of your Macbook. You can plug in a pair of headphones to avoid this from happening.


I guess I only noticed this after installing your software, since I infrequently have audio calls with my speakers on! Thanks!
